firestore/integration_test.go

```diff
@@ -1601,3 +1601,35 @@ func TestDetectProjectID(t *testing.T) {
 		t.Errorf("expected an error while using TokenSource that does not have a project ID")
 	}
 }
+
+func TestIntegration_ColGroupRefPartitions(t *testing.T) {
+	h := testHelper{t}
+	coll := integrationColl(t)
+	ctx := context.Background()
+
+	// Create a doc in the test collection so a collectionID is live for testing
+	doc := coll.NewDoc()
+	h.mustCreate(doc, integrationTestMap)
+
+	for _, tc := range []struct {
+		collectionID           string
+		expectedPartitionCount int
+	}{
+		// Verify no failures if a collection doesn't exist
+		{collectionID: "does-not-exist", expectedPartitionCount: 1},
+		// Verify a collectionID with a small number of results returns a partition
+		{collectionID: coll.collectionID, expectedPartitionCount: 1},
+		// TODO: could implement a scenario where a collectionID had a good
```

Comment:
Do other features or other firestore libraries use test fixtures like this? Is there any way around requiring a DB that large?

Reply:
No fixtures to my knowledge, though java does generate a collection with 2 * 128 + 127 docs. I could add this but had concerns that it would slow CI down for folks. There are techniques I could use to try and reuse existing docs so this wouldn't happen on our CI as long as the same project was used (fixed collection name).

Reply:
Added test but as a separate case